### PR TITLE
ddl: increase reorg delete limit.

### PR DESCRIPTION
**Pull request by disksing**
Due to the 1024 limit for each ddl job run, the drop speed is 512 keys per second in practice (ddl job run interval is 2s). 65536 should be more suitable.

@zimulala @coocood

### CODE REVIEW

File: ddl/table_test.go [hunk 1]
```diff
@@ -227,12 +227,12 @@ func (s *testTableSuite) TestTable(c *C) {
 	testCheckJobDone(c, d, job, false)
 
 	// Check background ddl info.
-	time.Sleep(testLease * 200)
+	time.Sleep(time.Second * 5)
```
Comment (zimulala) [inline ddl/table_test.go]: Sleep time is too long in test.

**Comment by disksing** [comment]
PTAL @zimulala

File: ddl/reorg.go [hunk 2]
```diff
@@ -171,17 +171,18 @@ func (d *ddl) isReorgRunnable(txn kv.Transaction, flag JobType) error {
 
 // delKeysWithPrefix deletes keys with prefix key in a limited number. If limit < 0, deletes all keys.
 func (d *ddl) delKeysWithPrefix(prefix kv.Key, jobType JobType, job *model.Job, limit int) (int, error) {
-	batch := limit
-	if batch == 0 {
-		return 0, nil
-	} else if batch < 0 {
-		batch = defaultBatchSize
-	}
 	delAll := limit < 0
```
Comment (coocood) [inline ddl/reorg.go]: `limitedDel := limit >=0` is more readable.

**Comment by coocood** [comment]
LGTM

File: ddl/table.go [hunk 3]
```diff
@@ -78,14 +78,16 @@ func (d *ddl) onCreateTable(t *meta.Meta, job *model.Job) error {
 	}
 }
 
+// Maximum number of keys to delete for each job run.
+var reorgDeleteLimit = 65536
```
Comment (zimulala) [inline ddl/table.go]: reorgDeleteLimit is only use for delReorgTable. Can you change it ?

**Comment by zimulala** [comment]
LGTM

**Pull request merged**